index.html

```diff
@@ -353,7 +353,7 @@ <h2>Securing JSON-LD Verifiable Credentials with SD-JWT</h2>
           to [[VC-DATA-MODEL-2.0]].
         </p>
         <p>
-          A [=conforming SD-JWT issuer implementation=] MUST use [[SD-JWT] to secure this media type.
+          A [=conforming SD-JWT issuer implementation=] MUST use SD-JWT to secure this media type.
```

Comment:
```suggestion
          A [=conforming SD-JWT issuer implementation=] MUST use [[SD-JWT]] to secure this media type.
```

Reply:
@TallTed Thanks. I did the same first. I'm not sure which way is preferable